platform/android/java_godot_wrapper.cpp

```diff
@@ -95,6 +97,15 @@ jobject GodotJavaWrapper::get_class_loader() {
 	}
 }
 
+jobject GodotJavaWrapper::get_application_context() {
+	if (_get_application_context) {
```

Comment:
@m4gr3d I might be accidentally re-introducing this here after rebasing everything. Was this one of the methods you moved in your PR?

Reply:
I don't see any match for `_get_application_context` in the 3.1 branch, nor any commit in the master branch removing it, so I guess this is actually a change you added for this PR specifically.

Whether you actually need/use it, that's for you to say :) @m4gr3d can maybe still comment on whether it's implemented the right way after the recent refactoring.

Reply:
`Activity` is a subclass of `Context`, so you can use the already defined `get_activity` method instead for places that need a `Context`.